src/provider.tsx

```diff
@@ -53,13 +56,31 @@ const OrderCloudProvider: FC<PropsWithChildren<IOrderCloudProvider>> = ({
   const [token, setToken] = useState<string | undefined>();
   const [authLoading, setAuthLoading] = useState(true);
 
+  if (openIdConnect?.enabled) {
+    if (
+      !openIdConnect ||
+      !openIdConnect.configs ||
+      openIdConnect.configs.length === 0
+    ) {
+      throw new Error(
+        "OpenID Connect is enabled, but no configurations were provided."
+      );
+    }
+
+    if (!openIdConnect.accessTokenQueryParamName) {
+      throw new Error(
+        "OpenID Connect is enabled, but accessTokenQueryParamName is missing."
+      );
+    }
+  }
+
   const handleLogout = useCallback(() => {
     queryClient.clear();
+    Tokens.RemoveAccessToken();
+    Tokens.RemoveRefreshToken();
     setIsAuthenticated(false);
     setIsLoggedIn(false);
     setToken(undefined);
```

Comment:
This was part of the troubleshooting I was doing with the race condition bug. It probably doesn't matter but setIsAuthenticated(false) will trigger `verifyToken` which relies on the values of the token so it makes sense that we clear out the tokens here first before setting `isAuthenticated` to false